src/lib/telemetry.js

```diff
@@ -16,8 +17,9 @@ const APP_NAME = 'debug-ping-view';
  * @returns {boolean} true if telemetry can be collected, else false.
  */
 function isTelemetryEnabled() {
-  // If the app is in development mode, we don't want to initialize Glean.
-  if (process.env.REACT_APP_ENV !== 'prod') {
+  // If the app environment is not defined (likely because of local development),
+  // then don't collect any data.
+  if (typeof(process.env.REACT_APP_ENV) !== "undefined") {
```

Comment:
```suggestion
  if (typeof(process.env.REACT_APP_ENV) === "undefined") {
```

I think this should be flipped. We don't want to collect by default when developing locally, which is when this will be undefined.

Reply:
Of course you're correct! Good catch!